packages/client/hmi-client/src/components/models/tera-typed-model-diagram.vue

```diff
@@ -60,7 +60,8 @@
 						/>
 					</section>
 					<tera-reflexives-toolbar
-						v-if="showReflexivesToolbar && model && strataModel"
+						v-if="model && strataModel"
+						v-show="showReflexivesToolbar"
```

Comment:
Separated `populateReflexiveOptions` into `v-show` instead of `v-if`. Since `showReflexivesToolbar` will switch between true and false  as user navigates forward or backward through the stratify drilldown (while the condition in v-if is unlikely to switch between true and false), this avoids some issues where `tera-reflexives-toolbar` would not be in the correct state after being unmounted and mounted again